src/group.rs

```diff
@@ -157,22 +162,35 @@ impl GroupOps for crate::IronOxide {
         rt.block_on(group_api::list(self.device.auth(), None))
     }
 
-    fn group_create(&self, opts: &GroupCreateOpts) -> Result<GroupMetaResult> {
+    fn group_create(&self, opts: &GroupCreateOpts) -> Result<GroupCreateResult> {
         let mut rt = Runtime::new().unwrap();
         let GroupCreateOpts {
             id: maybe_id,
             name: maybe_name,
             add_as_member,
+            members,
             needs_rotation,
         } = opts.clone();
 
+        let modified_add_as_member;
+        if members.contains(self.device.auth().account_id()) {
```

Comment:
`let modified_add_as_member = add_as_member || members.contains(self.device.auth().account_id())` is an easier way to write this.